test/Conversion/tosa-to-emitc.mlir

```diff
@@ -238,6 +238,12 @@ func @test_matmul(%arg0: tensor<1x14x19xf32>, %arg1: tensor<1x19x28xf32>) -> ten
 }
 
 // Reduce ops
+func @test_argmax(%arg0: tensor<13x21x3xi32>) -> tensor<21x3xi32> {
```

Comment:
NIT: This isn't a reduce op, thus the test should be inserted before the comment above (or the comment should be moved).

Reply:
To be more precise, `// Reduce ops` here only refers to ops that have `reuduce` in their name.